modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/CaseResult/CaseResultEditTest.tsx

```diff
@@ -45,13 +47,30 @@ const CaseResultEditTest = () => {
 
 	const {
 		caseResult,
+		caseResultsIssues,
```

Comment:
Add default value to avoid using `?`

```suggestion
		caseResultsIssues = [],
```